Fix build with -fno-common by declaring reoccuring symbols as static.

diff -upr ucspi-tcp6-1.05.orig/src/ip4_bit.c ucspi-tcp6-1.05/src/ip4_bit.c
--- ucspi-tcp6-1.05.orig/src/ip4_bit.c	2020-12-23 21:15:22.558014763 +0100
+++ ucspi-tcp6-1.05/src/ip4_bit.c	2020-12-23 21:16:37.385110722 +0100
@@ -8,7 +8,7 @@
 #define BITSUBSTITUTION
 
 stralloc sanumber = {0};
-char strnum[FMT_ULONG];
+static char strnum[FMT_ULONG];
 
 int getnumber(char *buf, int len, unsigned long *u)
 {
diff -upr ucspi-tcp6-1.05.orig/src/ip6_bit.c ucspi-tcp6-1.05/src/ip6_bit.c
--- ucspi-tcp6-1.05.orig/src/ip6_bit.c	2020-12-23 21:15:22.558014763 +0100
+++ ucspi-tcp6-1.05/src/ip6_bit.c	2020-12-23 21:16:37.388444075 +0100
@@ -7,7 +7,7 @@
 
 #define BITSUBSTITUTION
 
-char strnum[FMT_ULONG];
+static char strnum[FMT_ULONG];
 
 /**
  * This function converts a IPv6 address into its binary representation.
diff -upr ucspi-tcp6-1.05.orig/src/rblsmtpd.c ucspi-tcp6-1.05/src/rblsmtpd.c
--- ucspi-tcp6-1.05.orig/src/rblsmtpd.c	2020-12-23 21:15:22.561348115 +0100
+++ ucspi-tcp6-1.05/src/rblsmtpd.c	2020-12-23 21:16:37.391777428 +0100
@@ -136,7 +136,7 @@ void antirbl(char *base)
     decision = 1;
 }
 
-char strnum[FMT_ULONG];
+static char strnum[FMT_ULONG];
 static stralloc message;
 static stralloc info;
 
diff -upr ucspi-tcp6-1.05.orig/src/tcpclient.c ucspi-tcp6-1.05/src/tcpclient.c
--- ucspi-tcp6-1.05.orig/src/tcpclient.c	2020-12-23 21:15:22.568014820 +0100
+++ ucspi-tcp6-1.05/src/tcpclient.c	2020-12-23 21:16:37.391777428 +0100
@@ -67,7 +67,7 @@ static stralloc moreaddresses;
 
 static stralloc tmp;
 static stralloc fqdn;
-char strnum[FMT_ULONG];
+static char strnum[FMT_ULONG];
 char ipstr[IP6_FMT];
 
 char seed[128];
diff -upr ucspi-tcp6-1.05.orig/src/tcprules.c ucspi-tcp6-1.05/src/tcprules.c
--- ucspi-tcp6-1.05.orig/src/tcprules.c	2020-12-23 21:15:22.568014820 +0100
+++ ucspi-tcp6-1.05/src/tcprules.c	2020-12-23 21:16:37.395110779 +0100
@@ -71,7 +71,7 @@ void die_ip6(void) {
     strerr_die3x(100,SYNTAX,"invalid IPv6 address: on line: ",line.s);
 }
 
-char strnum[FMT_ULONG];
+static char strnum[FMT_ULONG];
 stralloc sanum = {0};
 
 void getnum(char *buf,int len,unsigned long *u) {
diff -upr ucspi-tcp6-1.05.orig/src/tcpserver.c ucspi-tcp6-1.05/src/tcpserver.c
--- ucspi-tcp6-1.05.orig/src/tcpserver.c	2020-12-23 21:15:22.568014820 +0100
+++ ucspi-tcp6-1.05/src/tcpserver.c	2020-12-23 21:16:37.398444132 +0100
@@ -58,8 +58,8 @@ char remoteip6str[IP6_FMT];
 static stralloc remotehostsa;
 char *remotehost = 0;
 
-char strnum[FMT_ULONG];
-char strnum2[FMT_ULONG];
+static char strnum[FMT_ULONG];
+static char strnum2[FMT_ULONG];
 
 static stralloc tmp;
 static stralloc fqdn;
